### ISSUE TITLE
Jump page ENUM not generating code

### ISSUE THREAD

**Issue by MickTheMechanic** [ImpulseAdventure/GUIslice-Builder#168]
**Describe the bug**
when the "jump page enum" field for a button is filled in the builder the code is not generated 

**Expected behavior**
If a page enum is entered into the jump page enum field, code should be generated for that button, as with previous versions of the builder.

**Comment by Pconti31** [comment]
@MickTheMechanic Yes, Text Button works fine but Image Button model is missing code:
```
    if (row > PROP_HEIGHT || row == PROP_ENUM)
      super.setModelChanged();
```
So the program is unaware of the jump change.  Due to a medical issue I'm unable to provide much support at the moment so it will be quite a while before i can do an update.  You do have two work arounds.  One, rename the ENUM and the code will be regenerated.  Two, delete and recreate the image button and add the jump at the same time.
Its likely to be quite a while before I check this repository again so if you need a quicker response you should post to Calvin's  repository.
Paul--

**Comment by MickTheMechanic** [comment]
No worries, I wasnt sure if you were aware of the issue.
Its not a problem for me, I have been manually adding the code to the .ino file. I will also try your workarounds.

Hope you get well soon

**Comment by Pconti31** [comment]
Fixed in Release 0.16.b010
